src/field_ground_effect.c

```diff
@@ -52,8 +170,19 @@ void FieldObjectUpdateMetatileBehaviors(struct MapObject *mapObj)
 
 void GetGroundEffectFlags_Reflection(struct MapObject *mapObj, u32 *flags)
 {
-    u32 reflectionFlags[2] = { 0x00000020, 0x00000010 };
```

Comment:
Why did you get rid of this and put `sReflectionFlags` outside of the function if it already matched like that?

Reply:
From the other files it seemed the accepted standard was to have `.rodata` data declared at the top of a C file, and not moving this would have forced me to scatter the rest of the data throughout the file as it would need to be below this declaration to match.

Reply:
The rodata doesn't need to be at the top of the file. Other files also have local arrays with other rodata objects defined further down. It seems more logical that the original code had these initialized inside the function rather than use a memcpy onto the stack.